cmd/provisioner-localpv/app/helper_blockdevice.go

```diff
@@ -52,7 +41,8 @@ const (
 
 // WaitForBDTimeoutCounts specifies the duration to wait for BDC to be associated with a BD
 // The duration is the value specified here multiplied by 5
-var WaitForBDTimeoutCounts int
+var WaitForBDTimeoutCounts = pflag.Int("bd-time-out", 12,
```

Comment:
Why was this flag moved here. It should be associated with the cobra command right?

Reply:
Moved it back.